frontend/src/components/core/ReportView/ReportView.tsx

```diff
@@ -83,23 +83,6 @@ function ReportView(props: ReportViewProps): ReactElement {
     componentRegistry,
   } = props
 
-  const renderBlock = (theme: Theme, node: BlockNode): ReactElement => (
-    <div className="block-container">
-      <ThemeProvider theme={theme}>
-        <Block
-          node={node}
-          reportId={reportId}
-          reportRunState={reportRunState}
-          showStaleElementIndicator={showStaleElementIndicator}
-          widgetMgr={widgetMgr}
-          widgetsDisabled={widgetsDisabled}
-          uploadClient={uploadClient}
-          componentRegistry={componentRegistry}
-        />
-      </ThemeProvider>
-    </div>
```

Comment:
Mostly a personal preference: Could just remove ThemeProvider out. I enjoy having renderBlocks especially since the component declaration is so lengthy and it'd be nice to only update it in one place if needed.